common/data_refinery_common/migrations/0019_sample_is_blacklisted.py

```diff
@@ -0,0 +1,49 @@
+# Generated by Django 2.1.5 on 2019-04-08 14:34
+
+import django.contrib.postgres.fields
+from django.db import migrations, models
+from django.core.paginator import Paginator
+
+from data_refinery_common.utils import load_blacklist
+
+def update_blacklisted_samples(apps, schema_editor):
+    '''Makes the platform_names on the Sample model human readable.
+    We were removing spaces from rnaSeq samples' platforms, which made
+    them unfriendly to users. This reverts the platform names to the
+    original that didn't have spaces in them.
+    We do this by munging the platform name on both the Sample object
+    and the list of rnaSeq platforms so that they are comparable, then
+    if they match we set the Sample.platform_name to what the list had
+    originally.
+    We can't import the Sample model directly as it may be a newer
+    version than this migration expects. We use the historical version.
+    '''
+    Sample = apps.get_model('data_refinery_common', 'Sample')
+
+    paginator = Paginator(Sample.objects.all(), 200)
```

Comment:
We can probably use a bigger page size right? IIRC bigger page sizes are more efficient but use more RAM right?